### PR TITLE
[CBRD-24032] The --compress(-z) option of backupdb utility is made to default option.

### PR DESCRIPTION
**Pull request by hornetmj**
https://jira.cubrid.org/browse/CBRD-24032

### Purpose
The --compress(-z) option of backupdb utility is made to default option.

### Implementation
* --no-compress option is introduced newly.
* --compress(-z) option is deprecated and is removed from the usages.

### Remarks
* --no-compress option works correctly.
* --compress(-z) option works correctly. (no error, backward compatibility)
* The usage doesn't show the description of the --compress(-z) option.
* The backupdb utility always compresses the backup data unless --no-compress option is used.

### CODE REVIEW

File: msg/zh_CN.utf8/utils.msg [hunk 1]
```diff
@@ -289,7 +289,7 @@ backupdb: 备份一个数据库.\n\
   -C, --CS-mode                客户端-服务器模式执行\n\
       --no-check               不检测数据库的一致性; 默认: 检测\n\
   -t, --thread-count=COUNT     线程数; 默认: 0(auto)\n\
-  -z, --compress               压缩备份卷; 默认: 不压缩\n\
+      --no-compress            不要压缩备份卷；默认值：压缩\n\
```
Comment (wrlawodms) [inline msg/zh_CN.utf8/utils.msg:292]:
```suggestion
      --no-compress            不要压缩备份卷; 默认值：压缩\n\
```
Sometimes, the Google translator modify ";" to "；" for JP and CN. It has to be ;

**Review by wrlawodms — APPROVED** [review]

File: msg/zh_CN.utf8/utils.msg (hunk 1 above)
Comment (wrlawodms) [reply msg/zh_CN.utf8/utils.msg:292]: I remember that `;` might be the line break, but I'm not sure. If it prints properly, you don't need to change it.

Comment (wrlawodms) [reply msg/zh_CN.utf8/utils.msg:292]: Oh, it's not. sorry.

File: src/executables/util_cs.c [hunk 2]
```diff
@@ -130,7 +130,7 @@ backupdb (UTIL_FUNCTION_ARG * arg)
   bool no_check = false;
   bool check = true;
   int backup_num_threads;
-  bool compress_flag;
+  bool no_compress_flag;
   bool sa_mode;
   bool separate_keys;
   FILEIO_ZIP_METHOD backup_zip_method = FILEIO_ZIP_NONE_METHOD;
```
Comment (mhoh3963) [inline src/executables/util_cs.c]: How about change assigned value of backup_zip_method to FILEIO_ZIP_LZ4_METHOD ?
If you are ok, please change assigned value of backup_zip_level to FILEIO_ZIP_LZ4_DEFAULT_LEVEL, together.

Comment (hornetmj) [reply src/executables/util_cs.c]: @mhoh3963 The --no-compress option doesn't work If we change them.

File: src/executables/util_cs.c [hunk 3]
```diff
@@ -190,7 +190,7 @@ backupdb (UTIL_FUNCTION_ARG * arg)
       goto print_backup_usage;
     }
 
-  if (compress_flag)
+  if (!no_compress_flag)
     {
       backup_zip_method = FILEIO_ZIP_LZ4_METHOD;
       backup_zip_level = FILEIO_ZIP_LZ4_DEFAULT_LEVEL;
```
Comment (hornetmj) [inline src/executables/util_cs.c]:
```suggestion
  if (no_compress_flag)
    {
      backup_zip_method = FILEIO_ZIP_NONE_METHOD;
      backup_zip_level = FILEIO_ZIP_NONE_LEVEL;
```

Comment (hornetmj) [reply src/executables/util_cs.c]: @mhoh3963 How about this? We can change the initial values of them if we change this code like this.

**Review by eido5 — APPROVED** [review]

File: src/executables/util_cs.c (hunk 3 above)
Comment (mhoh3963) [reply src/executables/util_cs.c]: It's good.

File: src/executables/util_cs.c (hunk 2 above)
Comment (mhoh3963) [reply src/executables/util_cs.c]: I think that you are better to change it with your below suggestion together.

**Review by mhoh3963 — APPROVED** [review]

**Comment by hornetmj** [comment]
@mhoh3963 I checked the failures of test_sql and they don't relate to this fix. So, merge this PR, please.

**Comment by mhoh3963** [comment]
@hornetmj  I merged.

**Pull request merged**